renderPage.js

```diff
@@ -174,7 +174,7 @@ export function renderListPage(Page) {
             tags={tags}
             closeModal={this.closeModal.bind(this)}
             updateFavorites={this.props.updateFavorites}
-            favorite={favorites.includes(modalClub.id)}
+            favorite={favorites.includes(modalClub.code)}
```

Comment:
What is the difference between `code` and `id`? Can you explain why we need this?

Reply:
the id is a numerical id assigned to each club, the code for each club is something like "pennlabs" or "penn-dems". I think it might be the url slug? Whatever the case, the favorites array contains string codes, not numerical ids.

Reply:
we good to yeet now?